ksqldb-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -574,7 +577,14 @@ private void printOverriddenProperties(final QueryDescription queryDescription)
 
   private void printQueryError(final QueryDescription query) {
     writer().println();
+
+    final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss.SSS");
     for (final QueryError error : query.getQueryErrors()) {
+      final Timestamp timestamp = new Timestamp(error.getTimestamp());
+      final String errorDate = timestamp.toInstant()
+          .atZone(ZoneId.systemDefault()).format(dateFormatter);
```

Comment:
Don't think we need the Sql Timestamp type here...

```suggestion
      final Instant ts = Instant.ofEpochMilli(error.getTimestamp());
      final String errorDate =ts
          .atZone(ZoneId.systemDefault()).format(dateFormatter);
```

Reply:
Done.